react/react1/week2/react1-week2/src/components/TodoList.jsx

```diff
@@ -0,0 +1,23 @@
+import TodoListRow from "./TodoListRow";
+const TodoList =({todos, setTodos}) =>{
+   
+    const deleteItem= (id)=>{
+   const newTodoArr  = todos.filter((todoItem) => todoItem.id !== id);
+   setTodos(newTodoArr)
+}
+
+return(
+<div>
+<ul>
+    {todos.map((todo)=>{
+        return(
+            <TodoListRow todo={todo.description} key={todo.id} id={todo.id} deleteItem={deleteItem}/>
```

Comment:
again, keep the naming cohesive
and the id and description props should be accessed in the component that will be used
```suggestion
            <TodoListItem todo={todo} key={todo.id} deleteItem={deleteItem}/>
```